tests/ert/unit_tests/gui/ertwidgets/test_copy_debug_info_button.py

```diff
@@ -0,0 +1,21 @@
+from pytestqt.qtbot import QtBot
+from qtpy.QtCore import Qt
+
+from ert.gui.simulation.run_dialog import CopyDebugInfoButton
+
+
+def test_copy_debug_info_button_alterates_text_when_pressed(qtbot: QtBot):
+    button_clicked = False
+
+    def on_click():
+        nonlocal button_clicked
+        button_clicked = True
+
+    button = CopyDebugInfoButton(on_click=on_click)
+    qtbot.addWidget(button)
+    initial_button_text = button.text()
+    assert initial_button_text == "Copy Debug Info"
```

Comment:
The test can probably fetch this string from `CopyDebugInfoButton` after you've extracted it?